tools/reviewer/job/sync/github_api.proto

```diff
@@ -51,45 +51,67 @@ message CreateIssueCommentRequest {
   message CreateIssueCommentRequestData { string body = 1; }
 }
 
-// https://developer.github.com/v3/pulls/reviews/#list-reviews-on-a-pull-request
-message ReviewsRequest {
+message CreateIssueCommentResponse { IssueComment issue_comment = 1; }
+
+// https://developer.github.com/v3/issues/comments/#edit-a-comment
+message EditIssueCommentRequest {
   string owner = 1;
   string repo = 2;
-  int64 number = 3;
+  int64 comment_id = 3;
+  EditIssueCommentRequestData request_data = 4;
+
+  message EditIssueCommentRequestData { string body = 1; }
 }
 
-message ReviewsResponse { repeated Review reviews = 1; }
+// https://developer.github.com/v3/issues/comments/#delete-a-comment
+message DeleteIssueCommentRequest {
+  string owner = 1;
+  string repo = 2;
+  int64 comment_id = 3;
+}
 
-// https://developer.github.com/v3/pulls/reviews/#get-comments-for-a-single-review
-message ReviewCommentsRequest {
+// https://developer.github.com/v3/pulls/comments/#create-a-comment
+message CreateReviewCommentRequest {
   string owner = 1;
   string repo = 2;
   int64 number = 3;
-  int64 review_id = 4;
+  CreateReviewCommentRequestData request_data = 4;
+
+  message CreateReviewCommentRequestData {
```

Comment:
Why do we split this into a sub-message and not add the fields directly under the `CreateReviewCommentRequest`?

Same for other messages, e.g `EditReviewCommentRequestData`.

Reply:
Ok, you're right. Forget my comment.